lib/arkana/kotlin_code_generator.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "erb"
+require "fileutils"
+require_relative "helpers/string"
+
+# Responsible for generating Kotlin source and test files.
```

Comment:
This generator isn't generating any test files 🙈 

Instead of updating the comment, could you implement unit test files like we're doing for Swift? 🙇 

They are kotlin unit test files generated when the source files are generated, that make sure the encoding/decoding is working as expected

Reply:
Yep, of course, will work on getting the tests generated too.